Makefile

```diff
@@ -14,7 +14,7 @@ lint: ## Runs the linters (including internal ones)
 	# internal analysis tools
 	go run ./internal/tool/analysis ./...;
 	# external analysis tools
-	golint ./...;
+	golint -set_exit_status ./...;
```

Comment:
Finally make golint fail if it finds issues, otherwise this won't crash locally, but only in the CI.